modules/appeals_api/app/controllers/appeals_api/schemas/shared_schemas_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class AppealsApi::Schemas::SharedSchemasController < AppealsApi::ApplicationController
+  skip_before_action :authenticate
+  before_action :check_schema_type
+
+  ACCEPTED_SCHEMA_TYPES = %w[
+    non_blank_string
+    address
+    date
+    phone
+    timezone
+  ].freeze
+
+  def show
+    render json: AppealsApi::Schemas::SchemaGetter.new(schema_type).call
+  end
+
+  private
+
+  def schema_type
+    @schema_type ||= params[:schema_type]
+  end
+
+  def check_schema_type
+    render json: invalid_schema_type_error, status: :not_found unless schema_type.in?(ACCEPTED_SCHEMA_TYPES)
+  end
+
+  def invalid_schema_type_error
+    {
+      title: 'not_found',
+      detail: I18n.t('appeals_api.errors.invalid_schema_type', schema_type: schema_type),
+      code: 'InvalidSchemaType',
+      status: '404',
+      source: { parameter: schema_type }
```

Comment:
Let's add a `meta` key with `available_schemas` to display the list of available options